src/main/java/blackjack/BlackJackApplication.java

```diff
@@ -0,0 +1,39 @@
+package blackjack;
+
+import java.util.List;
+
+import blackjack.controller.BlackJackController;
+import blackjack.dto.PlayerTurnDto;
+import blackjack.dto.TableStatusDto;
+import blackjack.service.BlackJackService;
+import blackjack.view.InputView;
+import blackjack.view.OutputView;
+
+public class BlackJackApplication {
```

Comment:
메인 클래스가 너무 복잡하군요.
아래와 같이 개선해보시기 바랍니다.

```suggestion
public class BlackJackApplication {
    public static void main(String[] args) {
        var blackJackGame = new BlackJackGame();
        blackJackGame.run();
    }
}
```